modules/integration/tests-integration/tests-backend/pom.xml

```diff
@@ -749,5 +749,10 @@
             <artifactId>org.wso2.carbon.logging.admin.stub</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Let's move all the RESP API tests to a new module (ex: tests-rest-api), without including them in tests-backend.